pure-pso/README.md

```diff
@@ -204,7 +204,7 @@ The following table lists the configurable parameters and their default values.
 | `flashblade.exportRules`                       | NFS Export Rules. Please refer the FlashBlade User Guide.                                                                                                  | ""                                            |
 | `flashblade.snapshotDirectoryEnabled`          | Enable/Disable FlashBlade snapshots                                                                                                                        | `false`                                       |
 | `orchestrator.name`                            | Orchestrator type, such as openshift, k8s                                                                                                                  | `k8s`                                         |
-| *`arrays`                                      | Array list of all the backend FlashArrays and FlashBlades                                                                                                  | must be set by user, see an example below     |
+| *`arrays`                                      | Array list of all the backend FlashArrays and FlashBlades                                                                                                  | may be set by user, see two examples below    |
```

Comment:
[nit] I'd just change it to "may be set by user, see examples below" (remove "two")